documentation/SolutionsRegistryFormat.md

```diff
@@ -49,6 +49,12 @@ The `settingsHandlers` block is unique and one of the most important blocks in t
         "options": {
             "filename": "${{environment}.APPDATA}\\Texthelp Systems\\RWSettings10.xml"
         },
+        "supportedSettings": [
```

Comment:
Simply specifying that it is a "hash with something non-falsy on the RHS" will be enough for this pull